### ISSUE TITLE
ffmpeg might need rebuild after libx265 update to 2.7

### ISSUE THREAD

**Issue by avih** [Alexpux/MINGW-packages#3734]
Recently running `ffmpeg` started spawning the following error message:
```
---------------------------
ffmpeg.exe - Entry Point Not Found
---------------------------
The procedure entry point x265_api_get_130 could not be located in the dynamic
link library T:\msys64\mingw64\bin\avcodec-58.dll. 
---------------------------
OK   
---------------------------
```

I tried building ffmpeg myself - with `--enable-libx265`, and it works fine.

I think ffmpeg needs a rebuild.

**Comment by lazka** [comment]
Fixed by fe6e18355652 (also in the repo)

**Comment by avih** [comment]
Confirmed fixed after an update. Thanks.
